manufacturing-ai-suite/wind-turbine-anomaly-detection/docs/user-guide/get-started.md

```diff
@@ -224,13 +222,13 @@ make status
   
       ![Anomaly prediction in grid active power](./_images/anomaly_power_prediction.png)
 
-## Bringing down the sample app
+## Bring down the sample app
 
   ```sh
   make down
   ```
 
-## Troubleshooting
+## Check logs
```

Comment:
In my view, troubleshooting is better prompting users to look at this when above stuff doesn't work as expected.